edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/IConnectionRegistry.cs

```diff
@@ -11,5 +11,7 @@ public interface IConnectionRegistry
     {
         Task<Option<IDeviceListener>> GetDeviceListenerAsync(IIdentity identity);
         Task<Option<IDeviceProxy>> GetDeviceProxyAsync(IIdentity identity);
+
```

Comment:
nit: should we remove this whitespace?

Reply:
I added because those two above belong to one group of operations and the one below to a different one